src/schema/formats.js

```diff
@@ -2,12 +2,23 @@ import parse from 'url-parse';
 
 const VALIDNUMRX = /^[0-9]{1,5}$/;
 
+// Firefox's version format is laxer than Chrome's, it accepts:
+// https://developer.mozilla.org/en-US/docs/Toolkit_version_format
+// We choose a slightly restricted version of that format (but still more
+// permissive than Chrome) to allow Beta addons, per:
+// https://developer.mozilla.org/en-US/Add-ons/AMO/Policy/Maintenance
+const TOOLKITVERSIONRX = /(a|alpha|b|beta|pre|rc)\d*$/;
```

Comment:
This is just a note to myself to change this to `TOOLKIT_VERSION_REGEX` before I merge. I didn't notice this until now, but no sense in more back and forth, as the patch is perfect otherwise.